### ISSUE TITLE
Error when reviewing python packages in certain images

### ISSUE THREAD

**Issue by adworjan** [ansible/ansible-navigator#1580]
##### ISSUE TYPE

- Bug Report

##### SUMMARY

When attempting to view the Python packages on certain images, such as registry.redhat.io/ansible-automation-platform-24/ee-supported-rhel8:latest, ansible-navigator will raise an error "Unexpected errors were encountered while running 'select'. Please log an issue with the log file contents." It will then return to the ansible-navigator images screen.

In the ansible-navigator.log the error is displayed

'ansible_navigator.actions._actions.run_interactive' Logging an uncaught exception
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ansible_navigator/actions/_actions.py", line 154, in run_interactive
    return run_action(app=app, interaction=interaction)
  File "/usr/lib/python3.9/site-packages/ansible_navigator/actions/select.py", line 41, in run
    app.steps.append(app.steps.current.select_func())  # add next
  File "/usr/lib/python3.9/site-packages/ansible_navigator/actions/images.py", line 319, in _build_image_content
    self._images.selected["python"]["details"],
KeyError: 'details'

##### ANSIBLE-NAVIGATOR VERSION

```
ansible-navigator 3.3.1
```

##### CONFIGURATION
ansible-navigator:
  app: welcome
  ansible:
    inventory:
      entries:
        - /home/adworjan/ansible/playbooks/AnsibleGit/inventory
  execution-environment:
    container-engine: podman
    image: towerpah.shadowman.dev/shadowmanee:20230715T050228
    pull:
      policy: missing
  logging:
    level: debug
  editor:
    command: code-server {filename}
    console: false
  ansible-builder:
    workdir: /home/adworjan/ansible/execution/

##### LOG FILE

'ansible_navigator.actions._actions.run_interactive' Logging an uncaught exception
Traceback (most recent call last):
  File "/usr/lib/python3.9/site-packages/ansible_navigator/actions/_actions.py", line 154, in run_interactive
    return run_action(app=app, interaction=interaction)
  File "/usr/lib/python3.9/site-packages/ansible_navigator/actions/select.py", line 41, in run
    app.steps.append(app.steps.current.select_func())  # add next
  File "/usr/lib/python3.9/site-packages/ansible_navigator/actions/images.py", line 319, in _build_image_content
    self._images.selected["python"]["details"],
KeyError: 'details'

##### STEPS TO REPRODUCE

1. Pull down the registry.redhat.io/ansible-automation-platform-24/ee-supported-rhel8:latest image
2. Run ansible-navigator images
3. Navigate into the ee-supported image
4. Navigate into Python Packages
5. Experience error

##### EXPECTED RESULTS

I would expect the list of all python packages, versions, and descriptions to appear

##### ACTUAL RESULTS

"Unexpected errors were encountered while running 'select'. Please log an issue with the log file contents." It will then return to the ansible-navigator images screen.

##### ADDITIONAL INFORMATION
This does not happen on all images (tested with the ee-creator image and did not receive an error) so the behavior is not consistent across all EE. But this was observed in the supported EE as well as a custom-built EE built with ansible-builder v3

**Comment by olivierlemasle** [comment]
I got the same error with `ansible-navigator 3.4.0`.

When browsing Python Packages for image `registry.redhat.io/ansible-automation-platform-24/ee-supported-rhel8:latest` (I reproduced the same issue with other images), I get the error

```
Unexpected errors were encountered while running 'select'. Please log an issue with the log file contents.
```

The log file contains the following error:

```
2023-07-23T16:11:35.654402+00:00 CRITICAL 'ansible_navigator.actions._actions.run_interactive' Subcommand 'select' encountered a fatal error.
2023-07-23T16:11:35.655151+00:00 ERROR 'ansible_navigator.actions._actions.run_interactive' Logging an uncaught exception
Traceback (most recent call last):
  File "/home/user/.local/lib/python3.11/site-packages/ansible_navigator/actions/_actions.py", line 154, in run_interactive
    return run_action(app=app, interaction=interaction)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/ansible_navigator/actions/select.py", line 41, in run
    app.steps.append(app.steps.current.select_func())  # add next
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/ansible_navigator/actions/images.py", line 319, in _build_image_content
    self._images.selected["python"]["details"],
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'details'
```

**Comment by shatakshiiii** [comment]
Fixed in #1587
